opfpush/src/main/java/org/onepf/opfpush/OPFPush.java

```diff
@@ -38,6 +44,12 @@ private OPFPush() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the {@link org.onepf.opfpush.OPFPushHelper} instance.
+     *
+     * @return {@link org.onepf.opfpush.OPFPushHelper} instance.
+     * @throws org.onepf.opfpush.exception.OPFPushException if {@code OPFPush} wasn't initialized.
```

Comment:
Yes, I've fixed this in the local branch and going to push this after fixing javadoc.